lib/src/event.dart

```diff
@@ -271,6 +271,20 @@ class Event {
     );
   }
 
+  factory Event.quick(
```

Comment:
I don't think I will keep `Event.quick` to keep as less code as possible in the library.

The `quicky` way to create a basic event is to use `Event.from` which only require the `kind`, `tags`, `content` and `privkey` supposing that everyone want to create a `kind=1` event is a huge assumption while in the context of nip4 we reduced the possibilities.

I can initialize `tags` with a empty array to reduce even more the input params to `kind` `content` and `privkey`